coralproject/talk
# CORL-2371 Fix Staff Badges

## What does this PR do?
This PR addresses https://vmproduct.atlassian.net/browse/CORL-2371 by fixing a bug causing staff badges to not be added to the appropriate comments. It also begins a fixture collection for server side testing

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
Create 2 sites under the same tenants, and a story for each site, as well as a member with the role of Staff, 
Observe that as a Staff user, you receive Staff badges on both stories.

Additionally: 
Create an Admin user, an Organization moderator, a Site moderator for the first site, a Site moderator for the second site and a general commenter.

Observe that:
1. Admin and Organization moderator users are given badges on both stories.
2. Site Moderator users receive badges on the story for which the site is in their moderation scopes, and not on the other story.
3. General Commenters do not receive badges anywhere.
 
## How do we deploy this PR?
This PR should not require any special actions to deploy.
